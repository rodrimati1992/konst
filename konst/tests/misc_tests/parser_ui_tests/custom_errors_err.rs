const _: usize = {
    let mut parser = konst::Parser::new("foo bar baz");
    parser.skip(3);
    konst::result::unwrap!(Err(parser.to_other_error(&"hello world!")))
};


fn main(){}