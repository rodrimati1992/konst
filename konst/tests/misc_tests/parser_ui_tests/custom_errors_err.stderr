error[E0080]: evaluation panicked:
              invoked `unwrap_ok` macro with an `Err` value: ParseError { start_offset: 3, end_offset: 11, direction: FromStart, kind: Other, extra_message: "hello world!" }
 --> tests/misc_tests/parser_ui_tests/custom_errors_err.rs:4:5
  |
4 |     konst::result::unwrap!(Err(parser.to_other_error(&"hello world!")))
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ evaluation of `_` failed here
  |
  = note: this error originates in the macro `konst::result::unwrap` (in Nightly builds, run with -Z macro-backtrace for more info)
